prow/git/v2/interactor_test.go

```diff
@@ -93,6 +93,118 @@ func TestInteractor_Clone(t *testing.T) {
 	}
 }
 
+func TestInteractor_CloneWithRepoOpts(t *testing.T) {
+	var testCases = []struct {
+		name          string
+		dir           string
+		from          string
+		remote        RemoteResolver
+		repoOpts      RepoOpts
+		responses     map[string]execResponse
+		expectedCalls [][]string
+		expectedErr   bool
+	}{
+		{
+			name:     "blank RepoOpts",
+			dir:      "/else",
+			from:     "/somewhere",
```

Comment:
Using strings "/dir" and "/from" improves readability of expected calls below.

Reply:
Will update to use "/mirrorclone" and "/secondaryclone" for even more readability.

Reply:
Done.